lib/node_modules/@stdlib/math/base/napi/binary/include/stdlib/math/base/napi/binary.h

```diff
@@ -570,6 +570,46 @@
 	};                                                                         \
 	NAPI_MODULE( NODE_GYP_MODULE_NAME, stdlib_math_base_napi_cf_c_init )
 
+/**
+* Macro for registering a Node-API module exporting an interface invoking a binary function accepting signed 64-bit integers and returning a double-precision floating-point number.
+*
+* @param fcn   binary function
+*
+* @example
+* static double fcn( const int_64 x, const int_64 y ) {
```

Comment:
```suggestion
* #include <stdint.h>
*
* static double fcn( const int_64 x, const int_64 y ) {
```

The header is missing from certain examples above. This can be addressed in a follow-up PR.